src/Agent/NewRelic/Agent/Core/AgentHealth/AgentHealthReporter.cs

```diff
@@ -536,6 +531,12 @@ public void ReportSupportabilityCollectorErrorException(string endpointMethod, T
 
         #region Log Events and Metrics
 
+        private ConcurrentDictionary<string, bool> loggingFrameworksReported = new ConcurrentDictionary<string, bool>();
```

Comment:
Minor nitpick: our coding style guidelines are to prefix private class members with an underscore, e.g. `_loggingFrameworksReported`